lib/tests/it/main.rs

```diff
@@ -1302,6 +1302,47 @@ async fn test_container_zstd() -> Result<()> {
     Ok(())
 }
 
+/// Test for zstd:chunked
+/// We need to handle the case of modified hardlinks into /sysroot
+#[tokio::test]
+async fn test_container_zstd_chunked() -> Result<()> {
+    let fixture = Fixture::new_v1()?;
+    let baseimg = &fixture.export_container().await?.0;
+    let basepath = &match baseimg.transport {
+        Transport::OciDir => fixture.path.join(baseimg.name.as_str()),
+        _ => unreachable!(),
+    };
+    let baseimg_ref = format!("oci:{basepath}");
+    let zstd_image_path = &fixture.path.join("zstd_chunked.oci");
+    let st = tokio::process::Command::new("skopeo")
+        .args([
+            "copy",
+            "--dest-compress-format=zstd:chunked",
```

Comment:
I'm ok with this but I think it'd be only a little bit more work to factor out a shared helper function for the two formats.